modules/quality/src/qualitybrisque.cpp

```diff
@@ -0,0 +1,429 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+/*
+Copyright (c) 2011 The University of Texas at Austin
+All rights reserved.
+
+Permission is hereby granted, without written agreement and without license or royalty fees, to use, copy,
+modify, and distribute this code (the source files) and its documentation for
+any purpose, provided that the copyright notice in its entirety appear in all copies of this code, and the
+original source of this code, Laboratory for Image and Video Engineering (LIVE, http://live.ece.utexas.edu)
+and Center for Perceptual Systems (CPS, http://www.cps.utexas.edu) at the University of Texas at Austin (UT Austin,
+http://www.utexas.edu), is acknowledged in any publication that reports research using this code. The research
+is to be cited in the bibliography as:
+
+1) A. Mittal, A. K. Moorthy and A. C. Bovik, "BRISQUE Software Release",
```

Comment:
Can you add also a reference about the [BRISQUE](https://ieeexplore.ieee.org/document/6272356) algorithm and the software release in the bibliography?

See for instance the [xobjdetect.bib](https://github.com/opencv/opencv_contrib/blob/master/modules/xobjdetect/doc/xobjdetect.bib) file. You can then cite the paper in the doxygen documentation, e.g. `@cite Mittal12`

Reply:
@catree  Thanks! I've added the bibliography and cited in both header and main C++ files.